source/includes/_catalog.md

```diff
@@ -803,16 +855,26 @@ List<PlanDetail> availableAddons = catalogApi.getAvailableAddons(baseProductName
 ```
 
 ```ruby
-base_product_name = 'Basic'
-KillBillClient::Model::Catalog.available_addons(base_product_name, 
-                                                options)
+catalog = KillBillClient::Model::Catalog
+
+base_product_name = 'Standard'
+
+available_addons = catalog.available_addons(base_product_name,
+                                            options)
 ```
 
 ```python
 catalogApi = killbill.CatalogApi()
         
 available_add_on_plans = catalogApi.get_available_addons(base_product_name='Basic')
 ```
+```javascript
+const catalogApi: killbill.CatalogApi = new killbill.CatalogApi(config);
+
+const availableAddOns: AxiosResponse<any> = await catalogApi.getAvailableAddons();
```

Comment:
Should'nt this be `const availableAddons: AxiosResponse<killbill.PlanDetail[], any>`